demo/customer-sdk-demo.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <title>Demo SDK use</title>
   <script src="../bower_components/socket.io-client/socket.io.js"></script>
-  <script src="bower_components/bringg-js-sdk/BringgSDK.js"></script>
-  <script src="bower_components/jquery/dist/jquery.min.js"></script>
+  <script src="../BringgSDK.js"></script>
```

Comment:
i pulled out the js code from the html file to a separate demoApp.js and also created a demo config file where you have a centralized place to edit the order /share/token you are working with in the demo.
i didn't find a good way to update the path to the sdk file itself
also removed the api key from the google maps api call. (apparently - you don't need it at all :-)